lib/logger/lib/logger.ex

```diff
@@ -439,16 +439,12 @@ defmodule Logger do
 
   @type backend :: :gen_event.handler()
   @type message :: IO.chardata() | String.Chars.t()
-  @type level :: :error | :info | :warn | :debug
+  @type level :: :logger.level() | :warn
   @type metadata :: keyword()
-  @levels [:error, :info, :warn, :debug]
+  @levels [:emergency, :alert, :critical, :error, :warning, :notice, :info, :debug]
```

Comment:
Just a tip: this PR will be easier to merge if we don't add new features. What I would do, if it works out, is to merge the integration and only one or two releases later add new features. Otherwise, if we add new features tied to `:logger` and then it turns out the integration doesn't work, we would be in a very hard spot.

Reply:
Ok, I haven't count that as a "new feature" as it do not add any new functions (except `Logger.warning/{1,2}`) . It only allows `Logger.log/{2,3}` and `Logger.bare_log/{2,3}` to use new logger levels. But I can roll this back or extract to separate PR.

Reply:
Well, we can call it an enhancement then but the above still applies. So we should roll this back for now. :) Tks.